### ISSUE TITLE
Summoners War Exporter

### ISSUE THREAD

**Issue by ematt321** [Xzandro/sw-exporter#167]
So I matched the server, ip, and all of the proxy settings and set it to manual. While it's listening to the port, I open summoners war on my iPhone 8 and it goes black screen and says "unable to connect to network." Is there a reason it won't let me open summoners war on the manual proxy? Also, could it have something to do with a firewall block initially never popping up for me to click allow?

**Comment by Xzandro** [comment]
Yes. The most common issues with SWEX are either entered the wrong IP / Port or a security software is blocking the access.

**Comment by ematt321** [comment]
could you or someone else tell me how to get rid of the firewall on summoners war for iPhone?

**Comment by ematt321** [comment]
> Yes. The most common issues with SWEX are either entered the wrong IP / Port or a security software is blocking the access.

My main issue seems to be with running SW on my phone once my phone matches the manual settings on swproxy. Also, on the proxy, my phone is unable to run any apps that require WiFi if that helps.

**Comment by Xzandro** [comment]
Well, the same still applies. Its probably some kind of security software (like firewalls) on your PC that block the access.
